### PR TITLE
docs: added comments to the readme

### PR DESCRIPTION
**Pull request by kushagraag**
Added a few comments in Readme file for this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ root of your application with the contents
 ```js
 const {join} = require('path');
 
-/**
+/*
```
Comment (OrKoN) [inline README.md]: I think this should not be changed

File: README.md [hunk 2]
```diff
@@ -143,9 +143,11 @@ The following example searches [developer.chrome.com](https://developer.chrome.c
 import puppeteer from 'puppeteer';
 
 (async () => {
+  // Setup browser
```
Comment (OrKoN) [inline README.md]:
```suggestion
  // Launch the browser and open a new blank page
```

File: README.md [hunk 3]
```diff
@@ -143,9 +143,11 @@ The following example searches [developer.chrome.com](https://developer.chrome.c
...
   const browser = await puppeteer.launch();
   const page = await browser.newPage();
 
+  // Put webpage address
```
Comment (OrKoN) [inline README.md]:
```suggestion
  // Navigate the page to a URL
```

**Review by OrKoN — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (kushagraag) [reply README.md:146]: yes, i guess your comment looks better

**Comment by kushagraag** [comment]
Your comments are better, i have changed comments as per your suggestions. 
Thanks.

File: README.md (hunk 1 above)
Comment (OrKoN) [inline README.md]:
```suggestion
/**
```

**Review by OrKoN — APPROVED** [review]

**Pull request merged**